xbmc/video/dialogs/GUIDialogVideoManagerVersions.h

```diff
@@ -28,7 +28,19 @@ class CGUIDialogVideoManagerVersions : public CGUIDialogVideoManager
   void SetVideoAsset(const std::shared_ptr<CFileItem>& item) override;
 
   static std::tuple<int, std::string> NewVideoVersion();
-  static bool ConvertVideoVersion(const std::shared_ptr<CFileItem>& item);
+  /*!
+   * \brief Convert a movie into a version of another movie. The function receives one of the
+   * movies as parameter and the user will choose with the GUI the other movie from the whole
+   * library. The mode of operation decides which role is played by the parameter movie and the
+   * user selected movie.
+   * \param[in] item Item to use as source or destination, depending on mode.
+   * \param[in] itemIsSource True: The item provided will be converted into a version, the user
+   * is asked to select from the whole library which movie to attach it to. False: The item provided
+   * is the movie that will have a new version attached to, and the user is asked to select from the
+   * whole library the movie that will be turned into a version.
+   * \return True for success and a version was created and attached, false otherwise.
+   */
+  static bool ConvertVideoVersion(const std::shared_ptr<CFileItem>& item, bool itemIsSource);
```

Comment:
Pretty please, can we avoid bool parameters. The reader of the calling code "never" knows what that f***k true or false means at the first glance (without looking at doxy/header). Can we please introduce an enum class with two speaking values and get rid of the bool param?

Reply:
Working on it...